rails_app/app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  primary_abstract_class
```

Comment:
Why add this when there is no customisation?

Reply:
This is autogenerated by the rails bootstrapper as many things in this PR are